src/ocl/utilocl.cpp

```diff
@@ -980,9 +980,9 @@ float transpose_CNHW2NCHW(const Handle& handle,
         auto&& kernels = handle.GetKernels(kernel_name, network_config);
         if(!kernels.empty())
         {
-            auto kernel  = kernels.front();
-            kernel.ldims = {{vld[0], vld[1], vld[1]}};
-            kernel.gdims = {{vgd[0], vgd[1], vgd[2]}};
+            auto kernel = kernels.front();
+            kernel.SetLocalDims(vld[0], vld[1], vld[2]);
```

Comment:
@zjing14 @JehandadKhan This code has been added in https://github.com/ROCm/MIOpen/pull/507. Please take a look at this line. I changed the last index thinking it was a typo, maybe I'm wrong.